File: benchmarks/fibonacci/fib.go
package main

import "fmt"

func fib(n int) int {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	n := 32
	result := fib(n)
	fmt.Printf("fib(%d) = %d\n", n, result)
}

File: benchmarks/fibonacci/fib.js
function fib(n) {
    if (n < 2) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

const n = 35;
const result = fib(n);
console.log(`fib(${n}) = ${result}`);
